towhee/__init__.py

```diff
@@ -76,7 +76,7 @@ def __call__(self, *args) -> List[Tuple]:
         return res
 
 
-def pipeline(task: str, fmc: FileManagerConfig = FileManagerConfig(), branch: str = 'main', force_download: bool = False):
+def pipeline(task: str, framework: str = 'pytorch', fmc: FileManagerConfig = FileManagerConfig(), branch: str = 'main', force_download: bool = False):
```

Comment:
1. `fmc` can be removed for the user does not care about it.
2. `task` in there, means pipeline's repo name, so I think we should change the name.
3.  `task` `branch` `framework` `foce_download` would be better than the current order.

Reply:
Fixed.